src/stream/src/executor/managed_state/join/mod.rs

```diff
@@ -193,21 +193,21 @@ impl<K: HashKey, S: StateStore> JoinHashMap<K, S> {
 
     /// Returns a mutable reference to the value of the key in the memory, if does not exist, look
     /// up in remote storage and return, if still not exist, return None.
-    #[allow(dead_code)]
-    pub async fn get<'a, 'b>(&'a mut self, key: &'b K) -> Option<&'a HashValueType<S>> {
-        let state = self.inner.get(key);
-        // TODO: we should probably implement a entry function for `LruCache`
-        match state {
-            Some(_) => self.inner.get(key),
-            None => {
-                let remote_state = self.fetch_cached_state(key).await.unwrap();
-                remote_state.map(|rv| {
-                    self.inner.put(key.clone(), rv);
-                    self.inner.get(key).unwrap()
-                })
-            }
-        }
-    }
+    // #[allow(dead_code)]
+    // pub async fn get<'a, 'b>(&'a mut self, key: &'b K) -> Option<&'a HashValueType<S>> {
+    //     let state = self.inner.get(key);
+    //     // TODO: we should probably implement a entry function for `LruCache`
+    //     match state {
+    //         Some(_) => self.inner.get(key),
+    //         None => {
+    //             let remote_state = self.fetch_cached_state(key).await.unwrap();
+    //             remote_state.map(|rv| {
+    //                 self.inner.put(key.clone(), rv);
+    //                 self.inner.get(key).unwrap()
+    //             })
+    //         }
+    //     }
+    // }
```

Comment:
Revert this?

Reply:
It's not used. And it triggers Rust compiler's false negative very often. So I commented it out. Anyone needs this can simply revert the comment.